builders/pallets-precompiles/pallets/proxy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Proxy Pallet
+description: Learn about the available extrinsics, storage methods, and constants in the proxy pallet on Moonbeam.
+keywords: proxy, substrate, moonbeam, polkadot
+---
+
+# The Proxy Pallet
+
+![Proxy Moonbeam Banner](/images/builders/pallets-precompiles/pallets/proxies-banner.png)
+
+## Introduction {: #introduction }
+
+Proxy accounts can be set up to perform a limited number of actions on behalf of users and are useful for keeping the underlying accounts safe. They allow users to keep their primary account secured safely in cold storage while enabling the proxy to actively perform functions and participate in the network with the weight of the tokens in the primary account.
+
+[Substrate's proxy pallet](https://wiki.polkadot.network/docs/learn-proxies){target=_blank} enables you to create proxy accounts, remove proxy accounts, make calls as a proxy account, and make announce proxy transactions. To add and remove proxy accounts, you can use the proxy precompile: a Solidity interface that can be interacted through the Ethereum API. For more information on how to use this contract, please refer to the [Proxy Precompile](/builders/pallets-precompiles/precompiles/proxy){target=_blank} guide.
```

Comment:
is make announce supposed to be anonymous? this is probably what you were referring to in the PR description about anonymous proxies but yeah since we don't allow those i agree that something should be added here

also i think the second sentence should say "...interacted with through the..."

Reply:
haha reading further and I see there is an "announce" extrinsic... I guess if this is announce I think we can probably remove the "make" beforehand

Reply:
Yeah the "make" was just a typo, thanks for catching it 😅